Documentation/contributing/development/dev_setup.rst

```diff
@@ -692,6 +692,54 @@ change:
     $ make generate-k8s-api
     $ git add go.mod go.sum vendor/
 
+Add/update a cilium/kindest-node image
+--------------------------------------
+
+Cilium might use its own fork of kindest-node so that it can use k8s versions
+that were not yet released by Kind maintainers.
+
+One other reason is the base image used on kindest-node, which has not been yet
+released: Debian Bookworm. Cilium requires Bookworm because the glibc version
```

Comment:
We'll never think of updating this when Bookworm is out, let's make it an example rather than a specific reason?

```suggestion
One other reason for using a fork is that the base image used on kindest-node may not have been release yet. For example, as of this writing, Cilium requires Debian Bookworm (yet to be released), because the glibc version
```